docs/src/@primer/gatsby-theme-doctocat/components/head.js

```diff
@@ -28,7 +29,7 @@ function Head(props) {
       <meta property="og:description" content={description} />
       <meta property="og:image" content={siteMetadata.imageUrl} />
       <meta property="twitter:card" content="summary_large_image" />
-      <script src={primerViewComponentsSrc}></script>
+      <script src={withPrefix(primerViewComponentsSrc)} type="text/javascript"></script>
```

Comment:
[from gatsby docs:](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/path-prefix/#add-the-path-prefix-to-paths-using-withprefix)

> For pathnames you construct manually, there’s a helper function, [withPrefix](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#add-the-path-prefix-to-paths-using-withprefix) that prepends your path prefix in production (but doesn’t during development where paths don’t need to be prefixed).